lab-adam/gulpfile.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+
+
+gulp.task('serverlaunch', function(){
+  gulp.src(`${__dirname}/test/*-test.js`, {read: false})
+  .pipe(mocha({reporter: 'nyan'}));
+});
+
+gulp.task('default', ['test']);
```

Comment:
It's a good practice to have a linting task! You could probably just create a default gulpfile and copy it into all of your projects (and build on it as we need more tasks).